Makefile

```diff
@@ -12,9 +12,10 @@ grunt:
 
 grunt-process-lists:
 	grunt execute:preProcessLists --browser=$(browser) --type=$(type)
+	grunt browserify:backgroundTest --browser=$(browser) --type=$(type)
```

Comment:
Adding it before the `moveout` step, otherwise the updated test files don't get copied over to the build folder.